<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pointtracker</class>
 <widget class="QMainWindow" name="pointtracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pointtracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>326</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="objectList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="setScaleBtn">
         <property name="text">
          <string>Set Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setOriginBtn">
         <property name="text">
          <string>Set Origin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPointsBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Add Points</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="pointsList">
       <property name="midLineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addObjectButton">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>62</height>
        </size>
       </property>
       <property name="text">
        <string>Add Trajectory</string>
       </property>
       <property name="iconSize">
        <size>
         <width>170</width>
         <height>159</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markDiscontinuityBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>62</height>
        </size>
       </property>
       <property name="text">
        <string>Mark Point as Discontinuity</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="deletePointButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Delete Point</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteObjectButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Delete Trajectory</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
